docs: update quickstart with container information (#39)
coreruleset/documentation: content/deployment/quickstart.md

```bash
systemctl restart httpd.service
```

## Using Containers

> Another quick option is to leverage our pre-packaged containers. You can use docker, podman, or any container compatible engine. Our images are published in the docker hub. The one you probably want is the `owasp/modsecurity-crs`: it already has everything to get you up and running quickly.

Reviewer: ```suggestion
Another quick option is to leverage our pre-packaged containers. You can use docker, podman, or any container compatible engine. Our images are published on [Docker Hub](https://hub.docker.com/r/owasp/modsecurity-crs). The one you probably want is the `owasp/modsecurity-crs`: it already has everything to get you up and running quickly.
```

> We already pre-package both Apache and Nginx Web servers with the corresponding ModSecurity engine. More engines (like Coraza) will come later.

Reviewer: ```suggestion
We already pre-package both Apache and Nginx web servers with the corresponding ModSecurity engine. More engines (like Coraza) will come later.
```

> So to protect your running Web Server, just get the image and use these configuration variables to make the WAF receive the requests and proxy your server.

Reviewer: ```suggestion
So to protect your running web server, just get the image and use these configuration variables to make the WAF receive the requests and proxy your server.
```

> This is an example `docker-compose` that can be used to pull the container images. You just need to change the `BACKEND` server so it points to your own server:

Reviewer: ```suggestion
This is an example `docker-compose.yaml`, that can be used to pull the container images. You just need to change the `BACKEND` server so it points to your own server:
```
